modules/logging-loki-storage-odf.adoc

```diff
@@ -0,0 +1,59 @@
+// Module is included in the following assemblies:
+// logging/cluster-logging-loki.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="logging-loki-storage-odf_{context}"]
+
+= ODF storage
+
+.Prerequisites
+
+* You have deployed Loki Operator.
+
+* You have deployed link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12[OpenShift Data Foundation].
+
+* You have configured your OpenShift Data Foundation cluster https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/managing_and_allocating_storage_resources/adding-file-and-object-storage-to-an-existing-external-ocs-cluster[for object storage]. 
+
+.Procedure
+
+. Create an ObjectBucketClaim in the `openshift-logging` namespace:
```

Comment:
```suggestion
. Create an `ObjectBucketClaim` in the `openshift-logging` namespace:
```

Reply:
I think this needs backticks since its part of an API object. I see it in the repo w/o backticks but its "object bucket claim" not "ObjectBucketClaim"